lib-es5/uploader.js

```diff
@@ -485,26 +487,27 @@ function call_api(action, callback, options, get_params) {
         result = parseResult(buffer, res);
         if (result.error) {
           result.error.http_code = res.statusCode;
-          deferred.reject(result.error);
+          handleUploadAPIError(callback, result, deferred, result.error);
```

Comment:
@strausr , if the `error` variable is always `result.error`, why not sending only the result object?

Reply:
in some places, we're just sending the result, like above.